wallet/src/main/scala/org/bitcoins/wallet/DLCWallet.scala

```diff
@@ -157,27 +170,28 @@ abstract class DLCWallet extends Wallet {
 
       dlcOfferDb = DLCOfferDb(
         eventId = dlc.eventId,
-        network = network,
         oraclePubKey = oracleInfo.pubKey,
         oracleRValue = oracleInfo.rValue,
         contractInfo = contractInfo,
-        penaltyTimeout = timeouts.penaltyTimeout,
         contractMaturity = timeouts.contractMaturity,
         contractTimeout = timeouts.contractTimeout,
         fundingKey = dlcPubKeys.fundingKey,
-        toLocalCETKey = dlcPubKeys.toLocalCETKey,
-        finalAddress = dlcPubKeys.finalAddress,
+        payoutAddress = dlcPubKeys.payoutAddress,
         totalCollateral = collateral,
         feeRate = feeRate,
         changeAddress = changeAddr
       )
 
       dlcInputs = utxos.map(outRef =>
-        DLCFundingInputDb(eventId = dlc.eventId,
-                          isInitiator = true,
-                          outPoint = outRef.outPoint,
-                          output = outRef.output,
-                          sigs = Vector.empty))
+        DLCFundingInputDb(
+          eventId = dlc.eventId,
+          isInitiator = true,
+          outPoint = outRef.outPoint,
+          output = outRef.output,
+          redeemScriptOpt = None, // todo negotiate these
```

Comment:
ping: Is this something simple to add here or is there work that needs to be done elsewhere? I'm not sure I understand what this is going to be

Reply:
It would need to be added the DLCOffer and DLCAccept